7/Constructor1/script.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+let mainTextAddY = document.getElementById("mainTextAddY");
+let mainTextAddN = document.getElementById("mainTextAddN");
+let mainText = document.getElementById("mainText");
+
+mainTextAddY.onclick = function(){
+    mainText.removeAttribute('disabled');
+};
+
+mainTextAddN.onclick = function(){
+    mainText.setAttribute('disabled', 'disabled');
+    mainText.style.opacity(0.5);
+};
+
+function getData(){
+    let newHTML;
+    //приём данных из формы
+    let title = title.value;// название
+    let description = description.value;// слоган
+    let menu = menu.value;// меню навигации
+    let LogoImg = logoImg.value;// логотип
+    let mainText = mainText.value;// основной текст
+    let colorTheme = colorTheme.value;// цветовая тема
+    let colorText = colorText.value;// цвет текста
+
+    document.body.style.backgroundColor = colorTheme;
+
+    //cоздание итоговой строки
+    newHTML = '<div id="main"><header><img src="' + LogoImg + '" alt="logo">';
+    newHTML += '<h1>' + title + '</h1>';
+    newHTML += '<h2>' + description + '</h2></header>';
+
+    //создание строки пунктов меню
+    let list = document.constructor.list;
```

Comment:
Please try to use ```document.querySelector```
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll

Reply:
I decided to use document.getElementsByName('list');
I don't know it's old style coding, but it works for sure))

Can you give an example of how to use the document.querySelector in the right way on this example???